tests/example_listings.py

```diff
@@ -7,31 +12,28 @@
 settings.load_profile("unit")
 
 
+@composite
+def requirements_list(draw, length, answers=False):
+    if answers:
+        elements = booleans() if length is not None else one_of(booleans(), none())
+    else:
+        elements = text(min_size=1, alphabet='abcdefgh', average_size=10)
```

Comment:
It does, but we're not touching the values too much either way, it'd be mostly testing the JSON schema validator. I think we're getting most of the benefit with the small alphabet giving us unpredictable values of varying length. We could throw in a few unicode characters there, but I wouldn't expect them to change much (mostly because unicode issues are often caused by how data is sent and received, which is different when running unit tests).